### PR TITLE
Documentation Updated

### PR DESCRIPTION
**Pull request by iitzIrFan**
**This update enhances the setup process and clarity of the ContribHub platform. Key changes include:**

- Tech Stack: Updated with Next.js, React, TypeScript, Tailwind CSS, Supabase, and GitHub API, with badges for quick reference.
- Setup Instructions: Clear steps for cloning the repo, installing dependencies, configuring environment variables, and running the app locally.
- Supabase Schema Import: Step-by-step guide to importing seed.sql into Supabase via the dashboard.
- Project Structure: Brief overview of key directories like /app, /components, /services, and /public.
- Contribution Guidelines: Emphasis on community contributions with a link to the Contributing Guidelines.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,30 +29,79 @@ ContribHub is a platform that connects developers with open-source projects that
...
+   ```bash
+   cp .env.example .env
+   ```
+
+4. **Start the development server**:
+   ```bash
+   npm run dev
+   ```
+   The application should now be running on `http://localhost:3000`.
+
```
Comment (swve) [inline README.md:72]: I think it's better to keep this in the [CONTRIBUTING.md](https://github.com/graphicmade/contribhub/blob/main/CONTRIBUTING.md) file

File: README.md [hunk 2]
```diff
@@ -28,30 +29,79 @@ ContribHub is a platform that connects developers with open-source projects that
...
+1. **Open the Supabase dashboard** and navigate to your project.
+2. Go to the **SQL** tab in the dashboard.
+3. **Copy the SQL content** from the `seed.sql` file located in the repository.
+4. **Paste the SQL** into the Supabase SQL editor and execute the query. This will set up the required database schema for ContribHub.
+5. After running the SQL script, your Supabase instance should be configured with the necessary tables and data.
+
+### Additional Configuration
+
+Once the schema is set up, make sure to update any remaining environment variables in your `.env` file to ensure the application connects properly to your Supabase instance.
```
Comment (swve) [inline README.md:85]: This is great, please put it in the [CONTRIBUTING.md](https://github.com/graphicmade/contribhub/blob/main/CONTRIBUTING.md) file

**Review by swve — CHANGES_REQUESTED** [review]
Finished reviewing this, some changes to do and it'll be ready to merge. 

Thanks.

File: README.md (hunk 2 above)
Comment (iitzIrFan) [reply README.md:85]: you want the mentioned changes in CONTRIBUTING.md file instead of README.md file right?

Comment (iitzIrFan) [reply README.md:85]: @swve review this :- I had moved the mentioned into CONTRIBUTING.md file from README.md file !

https://github.com/user-attachments/assets/23940586-36a8-456a-a254-a18fa2c9db56

**Pull request merged**